test/unit/math/prim/meta/size_test.cpp

```diff
@@ -0,0 +1,31 @@
+#include <stan/math/prim/meta.hpp>
+#include <gtest/gtest.h>
+#include <vector>
+
+TEST(MathMetaPrim, size_scalar) {
+  using stan::math::size;
+  EXPECT_EQ(1U, size(27.0));
+  EXPECT_EQ(1U, size(3));
+}
+
+TEST(MathMetaPrim, size_vector) {
+  using stan::math::size;
+  std::vector<double> x(10);
+  EXPECT_EQ(10U, size(x));
+}
```

Comment:
This is pre-existing, but would it be worth adding a test for `std::vector<Eigen::Matrix>`?

Reply:
Sure. Done.